src/main/java/net/atos/zac/app/zaken/converter/RESTZaakConverter.java

```diff
@@ -186,6 +188,9 @@ public Zaak convertToPatch(final RESTZaak restZaak) {
             zaak.setCommunicatiekanaal(communicatieKanaal.getUrl());
         }
         zaak.setZaakgeometrie(restGeometryConverter.convert(restZaak.zaakgeometrie));
+        if (restZaak.redenOpschorting != null) {
+            zaak.setOpschorting(new Opschorting(restZaak.indicatieOpschorting, restZaak.redenOpschorting));
```

Comment:
De gebruikte constructor is hier eigenlijk niet voor bedoeld. Deze is alleen bedoeld om aangeroepen te worden voor het deserializeren van json (zie @JsonbCreator annotatie). Wat mij betreft mag die constructor weg.